### ISSUE TITLE
Link on website to C++ compiler broken

### ISSUE THREAD

**Issue by kbarnhart** [landlab/landlab#531]
I've found that the link on the website to information about C compilers is broken.

http://landlab.readthedocs.io/en/latest/compilers_in_windows.html#compile-in-windows

@jennyknuth are you the right person to ask about this? or @mcflugen ?

**Comment by jennyknuth** [comment]
Do you know the correct link? The source file for those docs is in the
landlab docs repo and I can make that change.

**Comment by mcflugen** [comment]
@kbarnhart Where is the link that's broken?

**Comment by kbarnhart** [comment]
I found it broken here. 
https://github.com/landlab/landlab/wiki/Installing-Landlab-from-source-code-(%22developer-install%22)#2-installing-landlab-in-developer-mode

**Comment by mcflugen** [comment]
@kbarnhart, @jennyknuth: I think the link is correct. It seems the page just isn't being created on readthedocs. I'm not sure why but it definitely builds locally.

**Comment by mcflugen** [comment]
It looks like @jennyknuth removed the file with 4e26795 - or, rather, moved the file. It's now in `2014-docs`. We probably just need to move it back. @jennyknuth do you see a problem with this? Any reason it didn't make the cut into the new documentation?

**Comment by jennyknuth** [comment]
Yes, let's fix the link, we just need to put the information in the most
logical place.

The dividing line between the two sets of docs was mostly around where it
was easiest to maintain. Less code-specific, more general stuff like
installation and tutorials was put in the wiki, the code based stuff from
docstrings is in the readthedocs.

So it looks like this information about a developer install and a C++
compiler would maybe be best in the wiki?

Jenny

**Comment by mcflugen** [comment]
@jennyknuth I agree, this sounds like stuff that should go in the wiki. Could you add the contents of `2014-docs/compilers_in_windows.rst` to the wiki? Maybe as a separate page? Or maybe there already is a page it could be added to?

**Comment by nicgaspar** [comment]
@saisiddu The link on our developer install page (https://github.com/landlab/landlab/wiki/Installing-Landlab-from-source-code-(%22developer-install%22) ) goes to a c++ install for Python 2.7 (this web page : http://www.microsoft.com/en-us/download/details.aspx?id=44266 ). This seems like a weird thing to link to as probably most of our users will be using Python 3.7 these days. Should we just remove that link? Or is there a standard MS place to go to get the c++ compiler for a windows machine?

**Comment by saisiddu** [comment]
@nicgaspar: Hi Nicole, I do agree with you. I will update the link. I opened another issue previously and wanted to check if others face the same issue before editing the website.

**Comment by SiccarPoint** [comment]
A better option for a windows C++ compiler link would be very welcome!!
